src/app/TargetView/TargetSelect.tsx

```diff
@@ -217,7 +216,9 @@ export const TargetSelect: React.FC<TargetSelectProps> = ({ onSelect, simple, ..
             </Dropdown>
           </CardBody>
           <CardExpandableContent>
-            <CardBody>{!selected ? <NoTargetSelected /> : <SerializedTarget target={selected} />}</CardBody>
+            <CardBody id="serialized-target-details">
+              <SerializedTarget target={selected} />
+            </CardBody>
```

Comment:
I don't think this `<CardExpandableContent>` ever actually gets rendered:

https://www.patternfly.org/components/card#expandable-cards

so maybe it's worth just removing it from here. This looks like the only usage of `<SerializedTarget>`, too.

Reply:
Okayy removed :D